includes/admin/meta-boxes/views/html-order-items.php

```diff
@@ -143,15 +143,33 @@
 		</div>
 	<?php endif; ?>
 	<table class="wc-order-totals">
-		<?php if ( 0 < $order->get_total_discount() ) : ?>
+		<?php?>
```

Comment:
Empty PHP opening and closing tag. Maybe this can be removed or was the original intention to display the "Items Subtotal" only if a given criteria is met in an if statement that was not added?

Reply:
I think the Items subtotal is added with the intention to always be there, so removing the empty tags.

Reply:
Yes, I think so. cc @LevinMedia just to double check.

Reply:
👍 yep, thank you.